aten/src/ATen/Utils.cpp

```diff
@@ -31,6 +31,7 @@ Tensor empty_cpu(
     c10::optional<bool> pin_memory_opt,
     c10::optional<c10::MemoryFormat> memory_format_opt) {
   TORCH_INTERNAL_ASSERT_DEBUG_ONLY(device_or_default(device_opt).type() == DeviceType::CPU);
+  TORCH_INTERNAL_ASSERT_DEBUG_ONLY(layout_or_default(layout_opt) == Layout::Strided);
```

Comment:
Should we add in a corresponding test?

Reply:
no, asserts are supposed to be impossible to reach code, so you can't really test it